bin/compile

```diff
@@ -651,3 +639,8 @@ echo "TOOL=${TOOL}" > "${t}"
 if [ "${TOOL}" != "gb" ]; then
     echo "NAME=${name}" >> "${t}"
 fi
+
+info ""
+info "We placed the compiled binary in ./bin on your dyno."
+info "If you want to invoke it in your Procfile use ./bin/your-installed-directory-name"
```

Comment:
It would be more work, but something customized to the users app like this might be nicer:

```
Compiled binaries:
   ./bin/web
   ./bin/other-name
   ./bin/etc
```